diff -urN box2d-2.4.2.orig/CMakeLists.txt box2d-2.4.2/CMakeLists.txt
--- box2d-2.4.2.orig/CMakeLists.txt	2023-02-21 07:51:28.000000000 +0100
+++ box2d-2.4.2/CMakeLists.txt	2025-01-21 02:02:18.037836261 +0100
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.8)
 
 # https://cmake.org/cmake/help/latest/command/project.html
-project(box2d VERSION 2.4.1)
+project(box2d VERSION 2.4.2)
 
 # set(CMAKE_CONFIGURATION_TYPES "Debug;RelWithDebInfo" CACHE STRING "" FORCE)
 
