diff -urN lua-5.4.8.orig/src/liolib.c lua-5.4.8/src/liolib.c
--- lua-5.4.8.orig/src/liolib.c	2025-05-21 23:09:16.000000000 +0200
+++ lua-5.4.8/src/liolib.c	2025-06-09 12:26:15.379963138 +0200
@@ -119,9 +119,9 @@
 
 #include <sys/types.h>
 
-#define l_fseek(f,o,w)		fseeko(f,o,w)
-#define l_ftell(f)		ftello(f)
-#define l_seeknum		off_t
+#define l_fseek(f,o,w)		fseek(f,o,w)
+#define l_ftell(f)		ftell(f)
+#define l_seeknum		long
 
 #elif defined(LUA_USE_WINDOWS) && !defined(_CRTIMP_TYPEINFO) \
    && defined(_MSC_VER) && (_MSC_VER >= 1400)	/* }{ */
