diff -urN lua-5.5.0.orig/src/liolib.c lua-5.5.0/src/liolib.c
--- lua-5.5.0.orig/src/liolib.c	2025-12-15 11:44:39.000000000 +0100
+++ lua-5.5.0/src/liolib.c	2025-12-25 18:28:53.006592290 +0100
@@ -118,9 +118,9 @@
 
 #include <sys/types.h>
 
-#define l_fseek(f,o,w)		fseeko(f,o,w)
-#define l_ftell(f)		ftello(f)
-#define l_seeknum		off_t
+#define l_fseek(f,o,w)		fseek(f,o,w)
+#define l_ftell(f)		ftell(f)
+#define l_seeknum		long
 
 #elif defined(LUA_USE_WINDOWS) && !defined(_CRTIMP_TYPEINFO) \
    && defined(_MSC_VER) && (_MSC_VER >= 1400)	/* }{ */
