diff -urN lua-5.5.1/src/liolib.c lua-5.5.1.orig/src/liolib.c
--- lua-5.5.1/src/liolib.c	2026-08-04 21:42:02.749325087 +0200
+++ lua-5.5.1.orig/src/liolib.c	2026-07-24 12:23:06.000000000 +0200
@@ -118,9 +118,9 @@
 
 #include <sys/types.h>
 
-#define l_fseek(f,o,w)		fseeko(f,o,w)
-#define l_ftell(f)		ftello(f)
-#define l_seeknum		off_t
+#define l_fseek(f,o,w)		fseek(f,o,w)
+#define l_ftell(f)		ftell(f)
+#define l_seeknum		long
 
 #elif defined(LUA_USE_WINDOWS) && !defined(_CRTIMP_TYPEINFO) \
    && defined(_MSC_VER) && (_MSC_VER >= 1400)	/* }{ */
