diff -Naur project_ogg.orig/configure project_ogg/configure
--- project_ogg.orig/configure	2014-05-27 20:52:46.000000000 +0100
+++ project_ogg/configure	2016-03-26 20:56:21.583369845 +0000
@@ -10543,7 +10543,7 @@
   need_lib_prefix=no
   need_version=no
   library_names_spec='${libname}${release}${shared_ext}$versuffix ${libname}${release}${shared_ext}$major $libname${shared_ext}'
-  soname_spec='${libname}${release}${shared_ext}$major'
+  soname_spec='${libname}${release}${shared_ext}'
   finish_cmds='PATH="\$PATH:/sbin" ldconfig -n $libdir'
   shlibpath_var=LD_LIBRARY_PATH
   shlibpath_overrides_runpath=no
