diff -urN openal-soft-1.25.1.orig/CMakeLists.txt openal-soft-1.25.1/CMakeLists.txt
--- openal-soft-1.25.1.orig/CMakeLists.txt	2026-01-20 03:56:03.000000000 +0100
+++ openal-soft-1.25.1/CMakeLists.txt	2026-01-20 13:24:08.949857275 +0100
@@ -1779,8 +1779,6 @@
 
 set_target_properties(${IMPL_TARGET} PROPERTIES ${ALSOFT_STD_VERSION_PROPS}
     OUTPUT_NAME ${LIBNAME}
-    VERSION ${OpenAL_VERSION}
-    SOVERSION ${OpenAL_VERSION_MAJOR}
 )
 target_compile_definitions(${IMPL_TARGET}
     PRIVATE AL_BUILD_LIBRARY AL_ALEXT_PROTOTYPES  "ALC_API=${EXPORT_DECL}" "AL_API=${EXPORT_DECL}"
