diff -urN openal-soft-1.22.0.orig/CMakeLists.txt openal-soft-1.22.0/CMakeLists.txt
--- openal-soft-1.22.0.orig/CMakeLists.txt	2022-04-23 03:46:14.000000000 +0200
+++ openal-soft-1.22.0/CMakeLists.txt	2022-04-25 01:25:14.862904864 +0200
@@ -1392,10 +1392,7 @@
     ${OpenAL_SOURCE_DIR}/common
 )
 
-set_target_properties(${IMPL_TARGET} PROPERTIES OUTPUT_NAME ${LIBNAME}
-    VERSION ${LIB_VERSION}
-    SOVERSION ${LIB_MAJOR_VERSION}
-)
+set_target_properties(${IMPL_TARGET} PROPERTIES OUTPUT_NAME ${LIBNAME})
 target_compile_definitions(${IMPL_TARGET}
     PRIVATE AL_BUILD_LIBRARY AL_ALEXT_PROTOTYPES "ALC_API=${EXPORT_DECL}" "AL_API=${EXPORT_DECL}"
     ${CPP_DEFS})
