diff -ur project_openal.orig/CMakeLists.txt project_openal/CMakeLists.txt
--- project_openal.orig/CMakeLists.txt	2017-06-04 16:31:22.000000000 +0200
+++ project_openal/CMakeLists.txt	2017-06-29 00:11:39.688714613 +0200
@@ -1239,8 +1239,6 @@
     ENDIF()
 ENDIF()
 
-SET_TARGET_PROPERTIES(OpenAL PROPERTIES VERSION ${LIB_VERSION}
-                                        SOVERSION ${LIB_MAJOR_VERSION})
 SET_TARGET_PROPERTIES(OpenAL PROPERTIES OUTPUT_NAME ${LIBNAME})
 
 if(WIN32 AND NOT LIBTYPE STREQUAL "STATIC")
