diff -ur openal-soft-1.23.0.orig/CMakeLists.txt openal-soft-1.23.0/CMakeLists.txt
--- openal-soft-1.23.0.orig/CMakeLists.txt	2023-02-03 15:48:18.000000000 +0100
+++ openal-soft-1.23.0/CMakeLists.txt	2023-03-14 01:22:40.529391381 +0100
@@ -1424,10 +1424,7 @@
     ${OpenAL_SOURCE_DIR}/common
 )
 
-set_target_properties(${IMPL_TARGET} PROPERTIES OUTPUT_NAME ${LIBNAME}
-    VERSION ${LIB_VERSION}
-    SOVERSION ${LIB_MAJOR_VERSION}
-)
+set_target_properties(${IMPL_TARGET} PROPERTIES OUTPUT_NAME ${LIBNAME})
 target_compile_definitions(${IMPL_TARGET}
     PRIVATE AL_BUILD_LIBRARY AL_ALEXT_PROTOTYPES "ALC_API=${EXPORT_DECL}" "AL_API=${EXPORT_DECL}"
     ${CPP_DEFS})
