diff -urN openal-soft-1.21.0.orig/CMakeLists.txt openal-soft-1.21.0/CMakeLists.txt
--- openal-soft-1.21.0.orig/CMakeLists.txt	2020-11-04 11:00:35.000000000 +0100
+++ openal-soft-1.21.0/CMakeLists.txt	2020-11-05 11:36:36.817915348 +0100
@@ -1308,10 +1308,7 @@
     ${OpenAL_SOURCE_DIR}/common
 )
 
-set_target_properties(${IMPL_TARGET} PROPERTIES OUTPUT_NAME ${LIBNAME}
-    VERSION ${LIB_VERSION}
-    SOVERSION ${LIB_MAJOR_VERSION}
-)
+set_target_properties(${IMPL_TARGET} PROPERTIES OUTPUT_NAME ${LIBNAME})
 target_compile_definitions(${IMPL_TARGET}
     PRIVATE AL_BUILD_LIBRARY AL_ALEXT_PROTOTYPES ${CPP_DEFS})
 target_compile_options(${IMPL_TARGET} PRIVATE ${C_FLAGS})
