diff -urN openal-soft-1.22.2.orig/CMakeLists.txt openal-soft-1.22.2/CMakeLists.txt
--- openal-soft-1.22.2.orig/CMakeLists.txt	2022-06-27 10:33:53.000000000 +0200
+++ openal-soft-1.22.2/CMakeLists.txt	2022-06-29 00:29:22.939235800 +0200
@@ -1409,10 +1409,7 @@
     ${OpenAL_SOURCE_DIR}/common
 )
 
-set_target_properties(${IMPL_TARGET} PROPERTIES OUTPUT_NAME ${LIBNAME}
-    VERSION ${LIB_VERSION}
-    SOVERSION ${LIB_MAJOR_VERSION}
-)
+set_target_properties(${IMPL_TARGET} PROPERTIES OUTPUT_NAME ${LIBNAME})
 target_compile_definitions(${IMPL_TARGET}
     PRIVATE AL_BUILD_LIBRARY AL_ALEXT_PROTOTYPES "ALC_API=${EXPORT_DECL}" "AL_API=${EXPORT_DECL}"
     ${CPP_DEFS})
