diff -urN libpng-1.6.57.orig/CMakeLists.txt libpng-1.6.57/CMakeLists.txt
--- libpng-1.6.57.orig/CMakeLists.txt	2026-04-08 22:49:24.000000000 +0200
+++ libpng-1.6.57/CMakeLists.txt	2026-04-11 15:19:09.784534461 +0200
@@ -669,9 +669,7 @@
   list(APPEND PNG_LIBRARY_TARGETS png_shared)
   set_target_properties(png_shared
                         PROPERTIES OUTPUT_NAME "${PNG_SHARED_OUTPUT_NAME}"
-                                   DEBUG_POSTFIX "${PNG_DEBUG_POSTFIX}"
-                                   VERSION "${PNGLIB_SHARED_VERSION}"
-                                   SOVERSION "${PNGLIB_ABI_VERSION}")
+                                   DEBUG_POSTFIX "${PNG_DEBUG_POSTFIX}")
   if(UNIX AND AWK)
     if(HAVE_LD_VERSION_SCRIPT)
       set_target_properties(png_shared
