diff -urN libpng-1.6.44.orig/CMakeLists.txt libpng-1.6.44/CMakeLists.txt
--- libpng-1.6.44.orig/CMakeLists.txt	2024-09-12 22:44:44.000000000 +0200
+++ libpng-1.6.44/CMakeLists.txt	2024-09-18 02:03:37.816428545 +0200
@@ -692,9 +692,7 @@
   list(APPEND PNG_LIBRARY_TARGETS png_shared)
   set_target_properties(png_shared PROPERTIES
                         OUTPUT_NAME "${PNG_SHARED_OUTPUT_NAME}"
-                        DEBUG_POSTFIX "${PNG_DEBUG_POSTFIX}"
-                        VERSION "${PNGLIB_SHARED_VERSION}"
-                        SOVERSION "${PNGLIB_ABI_VERSION}")
+                        DEBUG_POSTFIX "${PNG_DEBUG_POSTFIX}")
   if(UNIX AND AWK)
     if(HAVE_LD_VERSION_SCRIPT)
       set_target_properties(png_shared PROPERTIES
