diff -ur project_png.orig/CMakeLists.txt project_png/CMakeLists.txt
--- project_png.orig/CMakeLists.txt	2017-06-28 20:46:03.000000000 +0200
+++ project_png/CMakeLists.txt	2017-06-29 00:08:35.018710429 +0200
@@ -767,15 +767,6 @@
   CREATE_SYMLINK(${PNGLIB_NAME}-config libpng-config)
 endif(NOT WIN32 OR CYGWIN OR MINGW)
 
-# SET UP LINKS
-if(PNG_SHARED)
-  set_target_properties(png PROPERTIES
-#   VERSION 16.${PNGLIB_RELEASE}.1.6.30
-    VERSION 16.${PNGLIB_RELEASE}.0
-    SOVERSION 16
-    CLEAN_DIRECT_OUTPUT 1)
-endif()
-
 # If CMake > 2.4.x, we set a variable used below to export
 # targets to an export file.
 # TODO: Use VERSION_GREATER after our cmake_minimum_required >= 2.6.2
