diff -urN libpng-1.6.55.orig/CMakeLists.txt libpng-1.6.55/CMakeLists.txt
--- libpng-1.6.55.orig/CMakeLists.txt	2026-02-09 21:02:20.000000000 +0100
+++ libpng-1.6.55/CMakeLists.txt	2026-02-10 01:55:25.395030259 +0100
@@ -666,9 +666,7 @@
   list(APPEND PNG_LIBRARY_TARGETS png_shared)
   set_target_properties(png_shared
                         PROPERTIES OUTPUT_NAME "${PNG_SHARED_OUTPUT_NAME}"
-                                   DEBUG_POSTFIX "${PNG_DEBUG_POSTFIX}"
-                                   VERSION "${PNGLIB_SHARED_VERSION}"
-                                   SOVERSION "${PNGLIB_ABI_VERSION}")
+                                   DEBUG_POSTFIX "${PNG_DEBUG_POSTFIX}")
   if(UNIX AND AWK)
     if(HAVE_LD_VERSION_SCRIPT)
       set_target_properties(png_shared
