diff -urN libpng-1.6.58.orig/CMakeLists.txt libpng-1.6.58/CMakeLists.txt
--- libpng-1.6.58.orig/CMakeLists.txt	2026-04-15 19:23:26.000000000 +0200
+++ libpng-1.6.58/CMakeLists.txt	2026-05-12 16:33:21.729081503 +0200
@@ -669,9 +669,7 @@
   list(APPEND PNG_LIBRARY_TARGETS png_shared)
   set_target_properties(png_shared
                         PROPERTIES OUTPUT_NAME "${PNG_SHARED_OUTPUT_NAME}"
-                                   DEBUG_POSTFIX "${PNG_DEBUG_POSTFIX}"
-                                   VERSION "${PNGLIB_SHARED_VERSION}"
-                                   SOVERSION "${PNGLIB_ABI_VERSION}")
+                                   DEBUG_POSTFIX "${PNG_DEBUG_POSTFIX}")
   if(UNIX AND AWK)
     if(HAVE_LD_VERSION_SCRIPT)
       set_target_properties(png_shared
