diff -ur project_webp.orig/configure project_webp/configure
--- project_webp.orig/configure	2017-01-30 21:41:24.000000000 +0100
+++ project_webp/configure	2017-04-09 19:45:54.921600199 +0200
@@ -10691,7 +10691,7 @@
   need_lib_prefix=no
   need_version=no
   library_names_spec='${libname}${release}${shared_ext}$versuffix ${libname}${release}${shared_ext}$major $libname${shared_ext}'
-  soname_spec='${libname}${release}${shared_ext}$major'
+  soname_spec='${libname}${release}${shared_ext}'
   finish_cmds='PATH="\$PATH:/sbin" ldconfig -n $libdir'
   shlibpath_var=LD_LIBRARY_PATH
   shlibpath_overrides_runpath=no
